prow.go

```diff
@@ -368,7 +368,8 @@ func (m *jobManager) newJob(job *Job) (string, error) {
 	pj.Annotations["ci-chat-bot.openshift.io/expires"] = strconv.Itoa(int(m.maxAge.Seconds() + launchDeadline.Seconds()))
 	prow.OverrideJobEnvVar(&pj.Spec, "CLUSTER_DURATION", strconv.Itoa(int(m.maxAge.Seconds())))
 	if job.Mode == "build" {
-		prow.SetJobEnvVar(&pj.Spec, "PRESERVE_DURATION", "12h")
+		// keep the built payload images around for a week
+		prow.SetJobEnvVar(&pj.Spec, "PRESERVE_DURATION", "168h")
```

Comment:
We just need to make sure this gets propagated to both `--delete-after` (hard TTL) and `--delete-when-idle` (soft TTL)

Reply:
at present it does not:

```
# import the initial release, if any
UNRESOLVED_CONFIG=$INITIAL ARTIFACTS=$(ARTIFACTS)/initial ci-operator \
  --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson \
  --image-mirror-push-secret=/tmp/push-auth \
  --gcs-upload-secret=/secrets/gcs/service-account.json \
  --namespace=$(NAMESPACE) \
  --delete-when-idle=$(PRESERVE_DURATION) \
  "${targets[@]}"
```

Reply:
cluster-bot doesn't appear to ever pass `--delete-after`, does ci-operator have a default for it?